microsoft-edge/web-platform/local-network-access.md

```diff
@@ -0,0 +1,130 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 08/18/2025
+---
+# Controlling Local Network Access in Microsoft Edge
+<!-- todo: improve "description:" above, after clarify intro -->
+
+Use the Local Network Access feature to grant or deny sites' access to your<!-- todo: who is "your"?  the (end-)user's? --> local network.
+<!-- todo:
+As an end-user, you can use the Local Network Access feature to grant or deny a website access to your local network.
+
+state the audience; clarify the audience and scenario.  Admins?  Devs?  End-users?
+what admin must configure the policies, for this to work?
+* `LocalNetworkAccessAllowedForUrls`
+* `LocalNetworkAccessBlockedForUrls`
+-->
+
+This feature delivers secure, privacy-respecting browsing experiences, while maintaining compatibility with modern web standards.
+
+This feature is introduced in Microsoft Edge 140.
+
+<!-- todo:
+See also:
+* []()
+-->
+
+
+<!-- ====================================================================== -->
+## What is Local Network Access?
+
+Local Network Access is a feature in Microsoft Edge that restricts a website's ability to send requests to servers on a user's local network.  This feature displays a prompt to end-users, requiring the end-user to grant or deny the site permission before these requests can be made.
+
+The ability to request this permission is restricted to secure contexts.
+
+
+<!-- ====================================================================== -->
+## How to Enable Local Network Access in Microsoft Edge
+
+Users<!-- todo: which type of users, in which scenario?  network admins?  devs?  end-users? --> can enable and manage Local Network Access permissions through the browser settings:
+
+<!-- todo: when steps are accurate, maybe condense: select x > y > z. -->
+
+1. In Microsoft Edge, select **Settings and more** (...) > **Settings**.
+
+   The **Settings** page opens.
+
+1. Select **Privacy, search, and services**.
+
+   The **Privacy, search, and services** page is displayed.
+
+1. Select **Site permissions**.
+
+   The **Site permissions** page is displayed.
+
+1. Select **All sites**.
+
+   The **All sites** page is displayed:
+
+   ![The All sites page in Edge Settings](./local-network-access-images/settings-all-sites.png) 
+
+   The Address bar shows `edge://settings/privacy/sitePermissions/allSitePermissions/localNetworkAccess`.<!-- todo: in some cases, might not end up there.  can comment-out, but keep this special url, for Maint -->
+
+1. For a listed site requests access to your local network, turn on the **[todo]** toggle:
+
+   ![Settings for a selected site](./local-network-access-images/settings-for-a-site.png)<!-- todo: improve png, this is Edge 139 not 140 -->
+
+
+<!-- ====================================================================== -->
+## Configuring policies
+<!-- todo:
+does the article's audience change from end-users to admins, here?
```

Comment:
who configured the policies?  only an Admin can?  same person as the rest of the article speaks to?  end-users?